libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager+Internal.h

```diff
@@ -25,6 +25,7 @@
 @property (nonatomic, assign) SFAppType appType;
 @property (nonatomic, weak, nullable) id<SalesforceSDKManagerFlow> sdkManagerFlow;
 @property (nonatomic, strong, nonnull) SFSDKSafeMutableDictionary<NSString *, UIViewController *> *snapshotViewControllers;
+@property (nonatomic, strong, nullable) UIViewController* nativeLoginViewController;
```

Comment:
Is the multiple window scenario covered? If not, could potentially make this into a dictionary like `snapshotViewControllers`, there's also the AuthHelper methods that take a scene parameter and can be called from a scene delegate

Reply:
I have not tested multi-window.  Because I am using the standard way of kicking off the login flow I would expect it to act the same way as our LoginViewController does today (for better or worse).  I'll add it to the considerations list so we don't loose track of it!